### PR TITLE
Configure prettier & eslint

### PR DESCRIPTION
**Pull request by mikestone14**
This PR:

* Configures Prettier & ESLint
* Runs Prettier & applies fixes
* Runs ESLint & applies fixes
* Adds Prettier & ESLint to CI

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,60 @@
+{
+  "extends": [
+    "eslint:recommended",
+    "plugin:react/recommended",
+    "prettier",
+    "universe"
```
Comment (Elkrival) [inline .eslintrc]: 👾

File: .eslintrc [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  },
+  "parserOptions": {
+    "requireConfigFile": false,
+    "babelOptions": {
+      "presets": ["@babel/preset-react"]
+    },
+    "ecmaFeatures": {
+        "jsx": true
+    },
+    "ecmaVersion": 12,
```
Comment (Elkrival) [inline .eslintrc]: I'm curious, why default to 12..? what do we think about latest...?
There are so many options.. all the way to 15..

File: .eslintrc [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    "no-unused-vars": ["error", { "varsIgnorePattern": "^_", "argsIgnorePattern": "^_" }],
+    "import/order": [
+      "error",
+      {
+        "alphabetize": {
+          "caseInsensitive": true,
+          "order": "asc"
+        },
+        "groups": ["builtin", "external", "internal"],
+        "newlines-between": "always",
```
Comment (Elkrival) [inline .eslintrc:48]: 👍

File: .eslintrc [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    "parserOptions": {
+      "project": "./tsconfig.json"
+    }
+  },
+  "plugins": [
+    "react",
+    "jest",
+    "testing-library",
+    "no-only-tests",
+    "sort-keys-fix",
```
Comment (Elkrival) [inline .eslintrc:33]: i'm all for alphabetizing , i'm curious will this sort keys alphabetically for us automatically or just throw an error..?

File: .eslintrc (hunk 1 above)
Comment (Elkrival) [inline .eslintrc]: What do you think about this..?
```suggestion
   "universe/node",
    "universe/web"
```

**Review by Elkrival — COMMENTED** [review]
curious about a few things..

File: package.json [hunk 5]
```diff
@@ -105,14 +108,25 @@
...
-    "eslint-plugin-react": "^7.24.0",
+    "eslint": "^8.57.0",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-config-universe": "^12.0.0",
+    "eslint-plugin": "^1.0.1",
+    "eslint-plugin-import": "^2.29.1",
+    "eslint-plugin-jest": "^27.9.0",
+    "eslint-plugin-no-only-tests": "^3.1.0",
+    "eslint-plugin-prettier": "^5.1.3",
+    "eslint-plugin-promise": "^6.1.1",
```
Comment (Elkrival) [inline package.json:119]: do we need to add this to the plugin array..?

File: .eslintrc (hunk 4 above)
Comment (mikestone14) [reply .eslintrc:33]: I think when we run the `lint:fix` command it will sort for us.

File: .eslintrc (hunk 1 above)
Comment (mikestone14) [reply .eslintrc]: what does this comment mean?

File: .eslintrc (hunk 2 above)
Comment (mikestone14) [reply .eslintrc]: Not sure, I grabbed this from another config. I'll set it to 15.

File: package.json (hunk 5 above)
Comment (mikestone14) [reply package.json:119]: https://github.com/AMP-SCZ/dpdash/pull/719/commits/f41bdf37ba018948b4fe335f58d8270451592b42

File: .eslintrc (hunk 1 above)
Comment (mikestone14) [reply .eslintrc]: https://github.com/AMP-SCZ/dpdash/pull/719/commits/f41bdf37ba018948b4fe335f58d8270451592b42

File: .eslintrc (hunk 2 above)
Comment (mikestone14) [reply .eslintrc]: https://github.com/AMP-SCZ/dpdash/pull/719/commits/f41bdf37ba018948b4fe335f58d8270451592b42

File: .eslintrc (hunk 1 above)
Comment (Elkrival) [reply .eslintrc]: Haha, the name universe stands out and there's nothing to be done code wise.

File: views/DeepDive.react.js [hunk 6]
```diff
@@ -50,18 +50,20 @@ class DeepDive extends Component {
       })
   }
   handleResize = () => {
-    this.setState({
-      width: window.innerWidth - this.state.marginWidth,
-      height: window.innerHeight - this.state.marginHeight,
-    })
+    this.setState((prevState) => ({
```
Comment (Elkrival) [inline views/DeepDive.react.js:53]: I wonder if we need this file...

**Review by Elkrival — APPROVED** [review]
Looks good

File: views/DeepDive.react.js (hunk 6 above)
Comment (mikestone14) [reply views/DeepDive.react.js:53]: Do we no longer render this?

Comment (Elkrival) [reply views/DeepDive.react.js:53]: This file is unused, we don't render it

**Pull request merged**